### DISCUSSION TITLE
[Closed] Skeleton View for loading

### DISCUSSION THREAD

**Discussion by tanmoy-sarkar-neu** [CommunityToolkit/Maui#193] [New Feature Discussions]
I like to propose skeleton loader which becomes a standard nowadays for modern apps. Here is a reference :

(https://stackoverflow.com/questions/60038109/xamarin-forms-collection-view-show-skeleton-while-data-loading?rq=1)

**Comment by pictos** [comment]
@tanmoy-sarkar-neu thanks for the proposal! Do you have any API spec for it or any idea on how you would like to use it in your API through XAML and C#?

**Reply by tanmoy-sarkar-neu** [reply]
hmm.. Looks like this nuget package https://github.com/sthewissen/Xamarin.Forms.StateSquid has already implemented skeleton view and has been merged to Xamarin Community Toolkit. However, I am not able to figure it out in Toolkit. Can you please check once?

**Reply by tanmoy-sarkar-neu** [reply]
Here it is : https://docs.microsoft.com/en-us/xamarin/community-toolkit/views/statelayout

**Reply by pictos** [reply]
With that, we don't need to add this feature?

**Reply by tanmoy-sarkar-neu** [reply]
Correct. Sorry for bothering. :) We can close this issue

**Reply by pictos** [reply]
@tanmoy-sarkar-neu you aren't bothering us. We are very happy when community members come up with new features! So anytime that you think that we should add some feature into the **Community**Toolkit don't hesitate to open a discussion

**Comment by CliffAgius** [comment]
Closed as answered.
